src/client-scopes/details/ScopeForm.tsx

```diff
@@ -52,6 +52,10 @@ export const ScopeForm = ({ clientScope, save }: ScopeFormProps) => {
   const displayOnConsentScreen = useWatch({
     control,
     name: "attributes.display.on.consent.screen",
+    defaultValue:
+      clientScope.attributes?.["display.on.consent.screen"] === undefined
+        ? "true"
+        : clientScope.attributes["display.on.consent.screen"],
```

Comment:
Perhaps we can use [nullish coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) here to fall back to `"true"`?
```suggestion
    defaultValue: clientScope.attributes?.["display.on.consent.screen"] ?? "true",
```

Reply:
Tried this one and works same way, thank you!

Reply:
Awesome!